packages/lib-utils/src/app/AppInitSDK.tsx

```diff
@@ -39,18 +37,26 @@ type AppInitSDKProps = {
  */
 const AppInitSDK: React.FC<AppInitSDKProps> = ({ children, configurations }) => {
   const { store, storeContextPresent } = useReduxStore();
+  const [canRender, setCanRender] = React.useState(false);
+
   const { appFetch, pluginStore, wsAppSettings, apiDiscovery = initAPIDiscovery } = configurations;
+
   React.useEffect(() => {
     try {
-      if (isUtilsConfigSet()) {
+      if (!isUtilsConfigSet()) {
         setUtilsConfig({ appFetch, wsAppSettings });
+        setCanRender(true);
       }
       apiDiscovery(store);
     } catch (e) {
       consoleLogger.warn('Error while initializing AppInitSDK', e);
     }
   }, [apiDiscovery, appFetch, store, wsAppSettings]);
 
+  if (!canRender) {
+    return null;
```

Comment:
https://coreos.slack.com/archives/C02EQERSV3N/p1646860045455809

Reply:
As a very temporary fix to the hook firing order this is fine